dotnet/src/dotnetframework/GxClasses/Middleware/GXHttpServices.cs

```diff
@@ -92,13 +92,13 @@ public override void webExecute()
 				}
 				else
 				{
-					this.SendResponseStatus(404, "Resource not found");
+					this.SendResponseStatus((int)HttpStatusCode.NotFound, "Resource not found");
 				}
 			}
 			catch (Exception ex)
 			{
-				SendResponseStatus(500, ex.Message);
-				HttpHelper.SetResponseStatusAndJsonError(context.HttpContext, "500", ex.Message);
+				SendResponseStatus((int)HttpStatusCode.InternalServerError, ex.Message);
```

Comment:
I think this code is unnecessary, since `SetError` already calls `SendResponseStatus`.

```suggestion
```